src/veh_interact.cpp

```diff
@@ -665,11 +671,11 @@ bool veh_interact::can_install_part() {
     if( !( use_aid || use_str ) ) {
         ok = false;
     }
-    msg << string_format( _( "> <color_%1$s>1 tool with %2$s %3$i</color> <color_white>OR</color> <color_%4$s>strength %5$i</color>" ),
+    msg << string_format( _( "> %1$s1 tool with %2$s %3$i</color> <color_white>OR</color> %4$sstrength %5$i</color>" ),
                           status_color( use_aid ), qual.obj().name.c_str(), lvl,
                           status_color( use_str ), str ) << "\n";
 
-    std::string install_color = string_format( "<color_%1$s>",  status_color( ok || g->u.has_trait( trait_DEBUG_HS ) ) );
+    std::string install_color = string_format( "%1$s",  status_color( ok || g->u.has_trait( trait_DEBUG_HS ) ) );
```

Comment:
The format string "%1$s" is a bit useless. In fact, it's completely useless. It will be replaced by the argument string and no additional formatting will happen.
```C++
string_format( "%1%s", whatever ) == whatever;
```

Reply:
Added translation notes.